cfbs/args.py

```diff
@@ -115,6 +115,31 @@ def get_arg_parser():
         help="Specify minimum version in 'cfbs generate-release-information'",
         dest="minimum_version",
     )
+    parser.add_argument(
+        "--to-json",
+        help="Output 'cfbs analyze' results to a JSON file; optionally specify the JSON's filename",
+        nargs="?",
+        const="analysis",
+        default=None,
+    )
+    parser.add_argument(
+        "--reference-version",
+        help="Specify version to compare against for 'cfbs analyze'",
+    )
+    parser.add_argument(
+        "--masterfiles-dir",
+        help="If the path given to 'cfbs analyze' contains a masterfiles subdirectory, specify the subdirectory's name",
+    )
+    parser.add_argument(
+        "--ignored-path-components",
+        help="Specify path components which should be ignored during 'cfbs analyze'",
```

Comment:
The help should probably mention that more than one component is specified with spaces like `--ignored-path-components one two three`.